frontend/.storybook/preview.js

```diff
@@ -0,0 +1,39 @@
+import { RouterContext } from "next/dist/shared/lib/router-context"
+// import { FormProvider, useForm } from "react-hook-form"
+// import { AuthProvider } from "../helpers"
+// import { addDecorator } from "@storybook/react"
+// import { initializeWorker, mswDecorator } from "msw-storybook-addon"
+
+// initializeWorker()
+// addDecorator(mswDecorator)
+
+// don't run in production or during build
+// if (process.env.NODE_ENV === "development" && typeof global.process === 'undefined') {
+//   const { worker } = require("../helpers/api/mocks/browser")
+//   worker.start()
+// }
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/
+    }
+  },
+  // decorators: [
```

Comment:
Our jest tests and the browser app use helpers/providers.tsx to provide auth context. To simplify story development maybe we can use that here as well as using a formprovider to give the form ctx?

Reply:
sounds great! and i'm sure that's how it's _supposed_ to work, because i've read _extensive_ documentation over the past week. have not had much success with global decorators so far, but maybe it was the package-lock? i'll give it another go

Reply:
AHA! Ok, have not solved anything, per se, but did finally notice the crucial difference between components that render successfully in storybook and those that blow up: `requireAuth`! Wrapping the story in the `AuthProvider` decorator will give it access to `useAuth`, but if the component is inside of `requireAuth` then something else needs to happen besides just providing context. No idea what, but I'm sure it's something really simple, once you know what you're doing...